assets/js/src/core/modules/asset/editor/types/folder/tab-manager/tabs/list/list-provider.tsx

```diff
@@ -225,26 +226,34 @@ export const ListSortingProvider = ({ children }: ListSortingProviderProps): Rea
 }
 
 export interface IListDataContext {
-  data: AssetGetGridApiResponse | undefined
+  data?: AssetGetGridApiResponse
+  errorData?: FetchBaseQueryError
```

Comment:
hmm, not sure if we should place the error information that are filter related in here.
The data context is holding the data that is shown in the grid.

Why are you putting this here instead of the filters?
What do you think about moving this to the filters?

Reply:
Or to be more precise to move this to the filter context. :P

Reply:
I was thinking about this and decided to proceed with this simplified logic as otherwise we would need to update each provider that uses specific filters. However, I agree that the alternative approach is ideologically better, so I've updated the logic accordingly. Thanks for highlighting it! :)